packages/react/src/components/Geo/LocationSearch/index.tsx

```diff
@@ -33,18 +34,33 @@ const LocationSearchControl = ({
   return null;
 };
 
-const LocationSearchStandalone = (props: LocationSearchProps) => {
+const LocationSearchStandalone = ({
+  onLoading = () => {},
+  onResult = () => {},
+  onResults = () => {},
+  onClear = () => {},
+  onError = () => {},
```

Comment:
Believe these will be recreated on every render leading to the `useEffect` on ln 47 running extraneously as they will not have a static reference. We should create a `noop` just below the imports in this file and replace the default values on these lines with it:

```
// example noop
function noop<T>(_: T) {
  return null;
}

//----//

const LocationSearchStandalone = ({
  onLoading = noop,
  onResult = noop,
  // ...other callbacks
}: LocationSearchProps) {
```

Reply:
Found that there is a noop in the lodash library, so used that.